runner/Dockerfile.dindrunner

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:20.04
 
 ARG TARGETPLATFORM
-ARG RUNNER_VERSION=2.287.1
+ARG RUNNER_VERSION=2.289.1
```

Comment:
I think we can safely omit this change from Dockerfiles now as we already bumped to more recent version of actions/runner which supports runner hooks.